Credit (#76)
pluginsGLPI/doc: source/approbalbymail/index.rst

Approbal by mail
================

Requirements (on-premise)
-------------------------

============ =========== ===========

> GLPI Version Minimum PHP Recommanded

Reviewer: ```suggestion
GLPI Version Minimum PHP Recommended
```

============ =========== ===========
10.0.x       8.1         8.2
============ =========== ===========

Install the plugin
------------------

.. note::
   `standard license <https://services.glpi-network.com/#offers>`__ (or higher) is required. This plugin is also available from the `Cloud <https://glpi-network.cloud/>`__.

> -  Go to the marketplace. Download and install the plugin **Approbal by mail**.

Reviewer: ```suggestion
-  Go to the marketplace. Download and install the plugin **ApprovalByMail**.
```

.. figure:: images/Approval_mail-1.png
   :alt:

Using the plugin
----------------

> Once activated, there are no specific settings to be made (apart from the notification layout, if you wish, via **setup > notifications > notification templates**. The template is **tickets validation**)

Reviewer: ```suggestion
Once activated, there are no specific settings to be made (apart from the notification layout, if you wish, via **Setup > Notifications > Notification templates**. The template is **Tickets validation**)
```

-  In a new ticket, enter the name of the person or group receiving the
   approval request.

.. figure:: images/Approval_mail-2.png
   :alt:

> An email will be received with the option of **validating** or **refusing** the request. This contains 3 links (if no changes have been made to the model).
> 
> -  the ticket reminder,
> -  the acceptance link,
> -  the refusal link

Reviewer: ```suggestion
An email will be received with the option of **Validating** or **Refusing** the request. This contains 3 links (if no changes have been made to the model).

-  The ticket reminder
-  The acceptance link
-  The refusal link
```
